.github/workflows/spread.yaml

```diff
@@ -6,8 +6,15 @@ on:
 
 jobs:
   spread-tests:
-    name: Run Spread tests
-    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        runner:
+          - name: X64
+            runs-on: ubuntu-latest
+          - name: ARM64
+            runs-on: [noble, ARM64, large]
```

Comment:
info: are these our self-hosted runners?

Reply:
Also, why the three runners though?

Reply:
it's not three runners. these are labels. it reads "give me any runner with the labels 'noble', 'ARM64' and 'large'". any runner matching those labels (self-hosted or not) would be selected